specification/paths/Shipments.json

```diff
@@ -5,6 +5,14 @@
     ],
     "summary": "Get all the shipments.",
     "description": "This endpoint retrieves shipments.",
+    "parameters": [
+      {
+        "name": "include",
+        "in": "query",
+        "type": "string",
+        "description": "Comma separated string of the relationships names you want to include the data of.<br> The relationships that can be included are shop, service, contract, service_options, status and files"
```

Comment:
First `relationships` should be singular. Missing period at the end of the sentence.